CNAME

```diff
@@ -1 +0,0 @@
-omemo.top
```

Comment:
@bascht: It seems that we need instructions on how to run it locally to eliminate the need to check it on someone elses github.